### PR TITLE
fix: should show 4xx status error html

### PR DESCRIPTION
**Pull request by fengmk2**
Don't redirect to error page url

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
原来的逻辑有问题，4xx 不应该当作 5xx 处理的。

File: app.js [hunk 1]
```diff
@@ -52,14 +52,21 @@ module.exports = app => {
...
+          }
+          this.status = 500;
+          this.body = `<h2>Internal Server Error, real status: ${status}</h2>`;
+          return;
         }
-        this.status = 500;
-        this.body = `<h2>Internal Server Error, real status: ${status}</h2>`;
+        // 4xx
+        this.status = status;
+        this.body = `<h2>${status} ${http.STATUS_CODES[status]}</h2>`;
```
Comment (dead-horse) [inline app.js]: 4xx 不跳 errorPage 了？

**Comment by dead-horse** [comment]
需要给 4xx 一个 clientErrorPage 或者增加自定义返回 html 的功能

**Comment by dead-horse** [comment]
1.4.5

**Pull request merged**